static/src/inline-svgs/contributions/coin.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="18" height="18" viewBox="5 5 18 18"><defs><circle id="a" cx="14" cy="14" r="9"/></defs><use xlink:href="#a" overflow="visible" fill="#FB0"/><clipPath id="b"><use xlink:href="#a" overflow="visible"/></clipPath><circle clip-path="url(#b)" fill="none" stroke="#FF9B0B" stroke-miterlimit="10" stroke-dasharray="2,2" cx="14" cy="14" r="7.615"/><g clip-path="url(#b)"><defs><path id="c" d="M11.64 14.604h.655v.593c0 .86-.296 1.237-.818 1.515v.89h4.902l.142-1.217h-3.478v-.03c.45-.33.726-.676.777-1.484l.01-.265h1.75v-.942h-1.687l.05-.665c.072-1.104.623-1.35 1.33-1.35.46 0 .798.07 1.145.194v-.962c-.39-.296-.89-.48-1.75-.48-1.36 0-2.373.777-2.373 2.404v.86h-.654v.94z"/></defs><clipPath id="d"><use xlink:href="#c" overflow="visible"/></clipPath><g clip-path="url(#d)"><defs><path id="e" d="M-55.676-2.755h986.54v542.77h-986.54z"/></defs><clipPath id="f"><use xlink:href="#e" overflow="visible"/></clipPath><path clip-path="url(#f)" fill="#C05303" d="M8.016 6.937h11.967v14.126H8.016z"/></g></g></svg>
```

Comment:
Please follow https://github.com/guardian/frontend/blob/master/docs/inline-svgs.md and get rid off all the `xmlns` things